Renegade X Launcher/VersionCheck.cs

```diff
@@ -145,11 +146,14 @@ public static void UpdateGameVersion()
                 };
                 InstructionsHash = versionData["game"]["instructions_hash"];
                 GamePatchPath = versionData["game"]["patch_path"];
-                //GamePatchUrls = versionData["game"]["patch_urls"].ToObject<string[]>();
-                GamePatchUrls = UpdateServerModelFactory.CreateUpdateServerModels(versionData["game"]["patch_urls"].ToObject<string[]>());
+
+                // Server URL's list & Friendly Names
+                foreach (var x in versionData["game"]["server_urls"].ToObject<dynamic>())
+                    RXPatcher.Instance.AddNewUpdateServer(x["url"].ToString(), x["friendly_name"].ToString());

```

Comment:
`"friendly_name"` should be `"name"`

Reply:
This needs the beta.json changed that's currently live on renegade-x.com

Reply:
I didn't realize we had this live. Done.